apps/web/src/pages/ApplicantDashboardPage/components/HeroCard.tsx

```diff
@@ -0,0 +1,90 @@
+import { Link } from "@gc-digital-talent/ui";
+import { ChevronDoubleRightIcon } from "@heroicons/react/24/solid";
+import * as React from "react";
+
+export type Color =
+  | "primary"
+  | "secondary"
+  | "tertiary"
+  | "quaternary"
+  | "quinary";
+
+export const colorMap: Record<Color, Record<string, string>> = {
+  primary: {
+    "data-h2-background-color":
+      "base:all(primary.light) base:dark:iap(primary)",
+    "data-h2-color":
+      "base:all(black) base:all:admin(white) base:all:iap(white)",
+  },
+  secondary: {
+    "data-h2-background-color":
+      "base(secondary) base:dark:admin(secondary.lighter) base:dark:iap(secondary.light)",
+    "data-h2-color":
+      "base:all(black) base:all:admin(white) base:all:iap(white)",
+  },
+  tertiary: {
+    "data-h2-background-color":
+      "base(tertiary.lighter) base:dark:iap(tertiary.light)",
+    "data-h2-color": "base:all(black) base:all:iap(white)",
+  },
+  quaternary: {
+    "data-h2-background-color":
+      "base(quaternary) base:dark:iap(quaternary.light)",
+    "data-h2-color": "base:all(black) base:all:iap(white)",
+  },
+  quinary: {
+    "data-h2-background-color": "base(quinary) base:dark:iap(quinary.light)",
+    "data-h2-color": "base:all(black) base:all:iap(white)",
+  },
+};
+
+export interface HeroCardProps {
+  color: Color;
+  title: string;
+  href: string;
+  children?: React.ReactNode;
+  ariaRole?: React.AriaRole;
+}
+
+export const HeroCard = ({
+  color,
+  title,
+  href,
+  children,
+  ariaRole,
+}: HeroCardProps) => {
+  return (
+    <div
+      data-h2-background-color="base:all(white)"
+      data-h2-radius="base(s)"
+      data-h2-shadow="base(s)"
+      data-h2-flex-grow="base(1)"
+      role={ariaRole}
+    >
+      <div
+        data-h2-display="base(block)"
+        data-h2-font-size="base(h6, 1) desktop(h5, 1)"
+        data-h2-font-weight="base(700)"
+        data-h2-padding="base(x1)"
+        data-h2-radius="base(s, s, 0px, 0px)"
+        {...{ ...colorMap[color] }}
+      >
+        <div data-h2-display="base(flex)">
+          <Link href={href}>{title}</Link>
+          <div data-h2-flex-grow="base(1)" />
+          <Link href={href}>
+            <ChevronDoubleRightIcon
+              style={{
+                width: "1em",
+                height: "1em",
+              }}
+            />
+          </Link>
+        </div>
```

Comment:
This will need to be reworked to be a single link to [contain the text and icon in the same link](https://www.w3.org/WAI/WCAG22/Techniques/html/H2#:~:text=The%20icon%20and%20text%20are%20contained%20in%20the%20same%20a%20element.). I think it could be done with:

```tsx
<Link ...>
  <span
    data-h2-flex-grid="base(center, x.4, 0)"
    data-h2-justify-content="base(space-beween)"
  >
    <span>{title}</span>
    <Icon />
  </span>
</Link>
```

Can you confirm the hydrogen syntax on this one @substrae ?

Reply:
Thanks, this worked!  But why the "x.4"?  Since it's justified it works with a gutter of 0, right?

Reply:
Oh, I have no idea. I just saw we were using that value in `IconLink.tsx` 😆

Reply:
Good catch on the rule though.  I ended up using the 0 - [fix header links](https://github.com/GCTC-NTGC/gc-digital-talent/pull/5857/commits/7728dbf5a0912be3a2e28ac82eeea4f2d60d1cb2)